Resources/ja.lproj/HockeySDK.strings

```diff
@@ -274,3 +274,6 @@
 "HockeyAuthenticationContactDeveloper" = "認証エラー。このAppのデベロッパに連絡してください。";
 "HockeyAuthenticationWrongEmailPassword" = "無効なメールアドレスまたはパスワードが入力されました。入力を確認し、やり直してください。";
 "HockeyAuthenticationAuthSecretMissing" = "HockeySDK “authorization secret” が設定されていません。このAppのデベロッパに連絡してください。";
+
+/* Accessibility strings */
+"HockeyAccessibilityHintRequired" = "必須";
```

Comment:
@Huan9 @TroubleMakerBen 
Please, find translations for key 'HockeyAuthenticationViewControllerPasswordPlaceholder' for HZ and JP. Translations are different. Is it a mistake?